src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 5;
+    private static final int MAX_SIDE = 100;
+    private static final int MAX_WIDTH = 200;
+    private static final int MAX_HEIGHT = 250;
+    private static final int MAX_BASE = 50;
+    private static final int MAX_RADIUS = 25;
+    private static final int MAX_TOP_BASE = 50;
+    private static final int MAX_BOTTOM_BASE = 100;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    private Figure getRandomSquare() {
+        int randomSide = random.nextInt(MAX_SIDE) + 1;
+        return new Square(colorSupplier.getRandomColor(), randomSide);
+    }
+
+    private Figure getRandomRectangle() {
+        int randomWidth = random.nextInt(MAX_WIDTH) + 1;
+        int randomHeight = random.nextInt(MAX_HEIGHT) + 1;
+        return new Rectangle(colorSupplier.getRandomColor(), randomWidth, randomHeight);
+    }
+
+    private Figure getRandomRightTriangle() {
+        int randomBase = random.nextInt(MAX_BASE) + 1;
+        int randomHeight = random.nextInt(MAX_HEIGHT) + 1;
+        return new RightTriangle(colorSupplier.getRandomColor(), randomBase, randomHeight);
+    }
+
+    private Figure getRandomCircle() {
+        int randomRadius = random.nextInt(MAX_RADIUS) + 1;
+        return new Circle(colorSupplier.getRandomColor(), randomRadius);
+    }
+
+    private Figure getRandomIsoscelesTrapezoid() {
+        int randomTopBase = random.nextInt(MAX_TOP_BASE);
+        int randomBottomBase = random.nextInt(MAX_BOTTOM_BASE);
+        int randomHeight = random.nextInt(MAX_HEIGHT);
+        return new IsoscelesTrapezoid(colorSupplier.getRandomColor(), randomTopBase,
+                randomBottomBase, randomHeight);
+    }

```

Comment:
private methods should go after public methods

Reply:
Ok, thx!